docs/use-events.md

```diff
@@ -18,23 +18,23 @@ In the Event Catalog, you’ll find the following information for every event:
     * expected throughput
     * development lifecycle phase
     * consuming systems
-* Schema: the overall structure of an event payload, described as an [Avro schema](https://docs.oracle.com/cd/E26161_02/html/GettingStartedGuide/avroschemas.html).
+* Schema: the overall structure of an event payload, described as an Avro schema. [Read more about Avro on the Oracle website](https://docs.oracle.com/cd/E26161_02/html/GettingStartedGuide/avroschemas.html).
 
 ![A table displaying all events in the Event Catalog on the CODE VA website. The headers are name, topic, lifecycle, domain, and actions. Filters allow content in the table to be filtered.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/18290306/c50c04e0-0f98-4901-81f8-0b3c9d709137)
 
 ![An event details page on the CODE VA website. The page gives an overview of one event, including the information listed above.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/18290306/c6f32b9f-475c-4426-aeeb-65bb2cf8d6a4)
 
 ## Producing and consuming events from the catalog
 
-Producers add events to the catalog and are responsible for keeping them up to date. You can read more about what to expect as a producer on the [producing events](produce-events.md) page.
+Producers add events to the catalog and are responsible for keeping them up to date. You can read more about what to expect as a producer on the [producing events page](produce-events.md).
```

Comment:
Happened to notice "producing events" in similar links was capitalized, like `Producing Events page`. Should we be consistent about if these page titles are capitalized or not? Same for "consuming events" below

Reply:
The page names have been capitalized.